prisma/schema.prisma

```diff
@@ -34,6 +35,12 @@ model user {
   passport   passport[]
 }
 
+model ceremonies {
+  ceremony_time     DateTime @db.Date @id
+  total_slots       Int
+  open_registration Boolean
+}
+
```

Comment:
sanity check, is this reflecting a migration previously applied in the id repo?

Reply:
My comment on this vanished :)
(No, but is now)